integration-tests/utils/integration-utils.js

```diff
@@ -192,6 +198,7 @@ const dismissNotifications = async page => {
 }
 
 const signIntoTerra = async (page, { token, testUrl }) => {
+  console.log('signIntoTerra ...')
```

Comment:
Is this intentional or left over from development?

Reply:
Intentional. It makes easier (hopefully for everyone) to follow test events when reading failed test log.